addons/web_editor/static/src/js/editor/odoo-editor/test/spec/tabs.test.js

```diff
@@ -55,8 +57,141 @@ describe('Tabs', () => {
                 contentAfter: `<p>a${oeTab('32.8906px')}a${oeTab('32.8906px')}[]b</p>`,
             });
         });
+        it('should insert tab characters at the beginning of two separate paragraphs', async () => {
+            await testEditor(BasicEditor, {
+                contentBefore: `<p>a[b</p>` +
+                               `<p>c]d</p>`,
+                stepFunction: editor => triggerEvent(editor.editable, 'keydown', { key: 'Tab'}),
+                contentAfterEdit: `<p>${oeTab('40px', false)}a[b</p>` +
+                                  `<p>${oeTab('40px', false)}c]d</p>`,
+                contentAfter: `<p>${oeTab()}a[b</p>` +
+                              `<p>${oeTab()}c]d</p>`,
+            });
+        });
+        it('should insert tab characters at the beginning of two separate indented paragraphs', async () => {
+            await testEditor(BasicEditor, {
+                contentBefore: `<p>${oeTab()}a[b</p>` +
+                               `<p>${oeTab()}c]d</p>`,
+                stepFunction: editor => triggerEvent(editor.editable, 'keydown', { key: 'Tab'}),
+                contentAfterEdit: `<p>${oeTab('40px', false)}${oeTab('40px', false, true)}a[b</p>` +
```

Comment:
`false, true` is kind of cryptic though...

Reply:
Would you prefer named options ? `oeTab('40px', { contenteditable: true, sizeFirst: false })` ?

Reply:
Well at that point I think it kind of renders the util useless doesn't it ? I mean you're like 10 characters away from writing the actual html, which would have been better IMO but you were all against me on this back when we merged `oeTab` in the first place... ^^'

Reply:
You're not wrong :-P